src/pages/workspace/reportFields/WorkspaceReportFieldsPage.tsx

```diff
@@ -64,13 +64,14 @@ function WorkspaceReportFieldsPage({
         if (!policy) {
             return [];
         }
-        return Object.values(filteredPolicyFieldList).map((reportField) => ({
+        return Object.entries(filteredPolicyFieldList).map(([reportKey, reportField]) => ({
```

Comment:
```suggestion
        return Object.entries(filteredPolicyFieldList).map(([reportFieldKey, reportField]) => ({
```

Reply:
I have renamed reportKey to reportFieldKey.